kafka-producer/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/KafkaUtils.java

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.connector;
+
+import kafka.javaapi.producer.Producer;
+import kafka.producer.ProducerConfig;
+import org.apache.axiom.om.OMOutputFormat;
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.transport.MessageFormatter;
+import org.apache.axis2.transport.base.BaseUtils;
+import org.apache.axis2.util.MessageProcessorSelector;
+import org.apache.commons.io.output.WriterOutputStream;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.core.axis2.Axis2MessageContext;
+import org.wso2.carbon.connector.core.util.ConnectorUtils;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.StringWriter;
+import java.util.Properties;
+
+public class KafkaUtils {
+
+    /**
+     * Read the value from the input parameter
+     */
+    public static String lookupTemplateParameter(MessageContext messageContext,
+                                                 String paramName) {
+        return (String) ConnectorUtils.lookupTemplateParamater(messageContext, paramName);
+    }
+
+    /**
+     * The ProducerConfig class encapsulates the values required for establishing the connection with brokers such as the broker list, message
+     * partition class, serializer class for the message, and partition key,etc.
+     */
+    public static Producer<String, String> getProducer(MessageContext messageContext) {
+
+        Axis2MessageContext axis2mc = (Axis2MessageContext) messageContext;
+        axis2mc.getAxis2MessageContext();
+        String brokers = (String) axis2mc.getAxis2MessageContext()
+                .getOperationContext().getProperty("kafka.brokerList");
+        String serializationClass = (String) axis2mc.getAxis2MessageContext()
+                .getOperationContext().getProperty("kafka.serializationClass");
```

Comment:
Yes.
All property names are moved to the constant variables.
Fixed in the PR - https://github.com/wso2/esb-connectors/pull/337